Parsers/!lmgtfy or !google search term.js

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
please remove the exclamation points from the filename

Reply:
Thanks @earlduque, I did not know about it, I've removed it